addon/controllers/building/edit/form.js

```diff
@@ -44,17 +50,25 @@ export default class BuildingFormController extends Controller {
     }
   }
 
+  @lastValue("fetchBuilding") building;
```

Comment:
Why do you need this? Couldn't this be handled by `fetchBuildingWork.lastSuccessful.value.building`?